api/user/index.ts

```diff
@@ -53,10 +54,13 @@ export async function getRoleAssignmentForUser(userInfo: UserInfo, req: any): Pr
 }
 
 export function getLocationInfo(roleAssignmentResponse: any): any [] {
-  const locationInfo = []
+  const locationInfo = [];
   roleAssignmentResponse.forEach(roleAssignment => {
     if (roleAssignment.attributes.primaryLocation) {
-      locationInfo.push(roleAssignment.attributes)
+      const isCaseAllocator = roleAssignment.authorisations ? roleAssignment.authorisations.includes(CASE_ALLOCATOR_ROLE) : false;
```

Comment:
isCaseAllocator = roleAssignment &&  roleAssignment.authorisations && roleAssignment.authorisations.includes(CASE_ALLOCATOR_ROLE);

Reply:
Also, could you please move this logic to separate util method and write some unit tests for it.